apps/web/src/components/Icon/Icon.stories.tsx

```diff
@@ -25,3 +26,9 @@ export const Home = () => (
     <HomeIcon />
   </Icon>
 );
+
+export const Search = (args: Props) => (
```

Comment:
If you add the correct typing to this story you won't need to import the `Props` type as it is inferred from reference.